### ISSUE TITLE
Missing package README file

### ISSUE THREAD

**Issue by mckennabarlow** [dotnet/aspnetcore#48392]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

As one of the top most installed NuGet packages, your Microsoft.AspNetCore.Authentication.JwtBearer package is extremely important to the developer ecosystem. The NuGet team has noticed that your package on NuGet.org is missing a README file. 

Please take a moment to add a README file to your package repository.
Our customer research indicates that one of the top problems that package consumers face is insufficient package documentation such as README files. Adding a README file to your package will help users quickly understand what the package is and what it does. Since your README will be the first impression for users when they view your package on NuGet.org, it is crucial for authors to write and include high-quality READMEs for their packages.

### Describe the solution you'd like

Please add a README file to your package so that users can understand at a glance what the package contains and how to use it.

There is no specific format or structure to a good README – instead it’s about effectively communicating the purpose and functionality of your package to potential users. If you are unsure of where to start, check out our [blog post](https://devblogs.microsoft.com/nuget/write-a-high-quality-readme-for-nuget-packages/) for some README best practices and a template file.
If you are new to NuGet README, learn more about [how to add a README to your NuGet Package](https://devblogs.microsoft.com/nuget/add-a-readme-to-your-nuget-package/).

### Additional context

Your feedback is very important to us. If you experience any issue using READMEs or have ideas for improvements – please feel free to contact us by replying to this issue

### README Template
[https://raw.githubusercontent.com/lyndaidaii/NuGetReadMeTemplate/main/README.md](https://raw.githubusercontent.com/lyndaidaii/NuGetReadMeTemplate/main/README.md)

**Comment by Tratcher** [comment]
@mckennabarlow let's make a checklist of all our packages in this issue rather than filing an issue for each package.
- [ ] Microsoft.AspNetCore.Authentication.JwtBearer
- [ ] Microsoft.AspNetCore.Authentication.OpenIdConnect
- [ ] Microsoft.AspNetCore.Mvc.NewtonsoftJson
- [ ] Microsoft.AspNetCore.Identity.EntityFrameworkCore
- [ ] ...?

**Comment by amcasey** [comment]
FYI @halter73 for auth.

**Comment by halter73** [comment]
Is `area-security` the best label for this? I'm not sure it makes sense if it includes adding READMEs to other packages like Microsoft.AspNetCore.Mvc.NewtonsoftJson. Is @Tratcher's list comprehensive?

Do we have a good label for general packaging cleanup? `area-infrastructure`? @mkArtakMSFT

**Comment by Tratcher** [comment]
I know there are a bunch of other packages, I just wasn't sure the best way to get a comprehensive list.

**Comment by mckennabarlow** [comment]
Hi there, these are the packages owned by aspnet that I have as lacking a README file. I know this is a super long list -- let me know how I can best assist your team in addressing these missing files.

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/mcbarlow/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/mcbarlow/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



Package Name | Package Version
-- | --
ajaxcontroltoolkit | 20.1.0
dotnet-ef | 7.0.4
entityframework | 6.4.4
jquery | 3.6.4
microsoft.aspnet.identity.core | 2.2.3
microsoft.aspnet.identity.entityframework | 2.2.3
microsoft.aspnet.identity.owin | 2.2.3
microsoft.aspnet.mvc | 5.2.9
microsoft.aspnet.signalr | 2.4.3
microsoft.aspnet.signalr.core | 2.4.3
microsoft.aspnet.web.optimization | 1.1.3
microsoft.aspnet.webapi | 5.2.9
microsoft.aspnet.webapi.client | 5.2.9
microsoft.aspnet.webapi.core | 5.2.9
microsoft.aspnet.webapi.cors | 5.2.9
microsoft.aspnetcore | 2.2.0
microsoft.aspnetcore.app | 2.2.8
microsoft.aspnetcore.app.ref | 7.0.4
microsoft.aspnetcore.authentication | 2.2.0
microsoft.aspnetcore.authentication.google | 7.0.4
microsoft.aspnetcore.authentication.jwtbearer | 7.0.4
microsoft.aspnetcore.authentication.negotiate | 7.0.4
microsoft.aspnetcore.authentication.openidconnect | 7.0.4
microsoft.aspnetcore.components.authorization | 7.0.4
microsoft.aspnetcore.components.webassembly | 7.0.4
microsoft.aspnetcore.cors | 2.2.0
microsoft.aspnetcore.diagnostics.entityframeworkcore | 7.0.4
microsoft.aspnetcore.hosting | 2.2.7
microsoft.aspnetcore.http | 2.2.2
microsoft.aspnetcore.http.abstractions | 2.2.0
microsoft.aspnetcore.http.extensions | 2.2.0
microsoft.aspnetcore.http.features | 5.0.17
microsoft.aspnetcore.identity | 2.2.0
microsoft.aspnetcore.identity.entityframeworkcore | 7.0.4
microsoft.aspnetcore.identity.ui | 7.0.4
microsoft.aspnetcore.mvc | 2.2.0
microsoft.aspnetcore.mvc.core | 2.2.5
microsoft.aspnetcore.mvc.newtonsoftjson | 7.0.4
microsoft.aspnetcore.mvc.razor.runtimecompilation | 7.0.4
microsoft.aspnetcore.mvc.testing | 7.0.4
microsoft.aspnetcore.openapi | 7.0.4
microsoft.aspnetcore.session | 2.2.0
microsoft.aspnetcore.signalr | 1.1.0
microsoft.aspnetcore.signalr.client | 7.0.4
microsoft.aspnetcore.spaservices.extensions | 7.0.4
microsoft.aspnetcore.staticfiles | 2.2.0
microsoft.aspnetcore.testhost | 7.0.4
microsoft.codedom.providers.dotnetcompilerplatform | 3.6.0
microsoft.data.sqlite | 7.0.4
microsoft.data.sqlite.core | 7.0.4
microsoft.entityframeworkcore | 7.0.4
microsoft.entityframeworkcore.abstractions | 7.0.4
microsoft.entityframeworkcore.analyzers | 7.0.4
microsoft.entityframeworkcore.design | 7.0.4
microsoft.entityframeworkcore.inmemory | 7.0.4
microsoft.entityframeworkcore.proxies | 7.0.4
microsoft.entityframeworkcore.relational | 7.0.4
microsoft.entityframeworkcore.sqlite | 7.0.4
microsoft.entityframeworkcore.sqlite.core | 7.0.4
microsoft.entityframeworkcore.sqlserver | 7.0.4
microsoft.entityframeworkcore.sqlserver.design | 1.1.6
microsoft.entityframeworkcore.tools | 7.0.4
microsoft.entityframeworkcore.tools.dotnet | 2.0.3
microsoft.extensions.caching.memory | 7.0.0
microsoft.extensions.configuration | 7.0.0
microsoft.extensions.configuration.abstractions | 7.0.0
microsoft.extensions.configuration.binder | 7.0.4
microsoft.extensions.configuration.environmentvariables | 7.0.0
microsoft.extensions.configuration.fileextensions | 7.0.0
microsoft.extensions.configuration.json | 7.0.0
microsoft.extensions.configuration.usersecrets | 7.0.0
microsoft.extensions.dependencyinjection | 7.0.0
microsoft.extensions.dependencyinjection.abstractions | 7.0.0
microsoft.extensions.hosting | 7.0.1
microsoft.extensions.hosting.abstractions | 7.0.0
microsoft.extensions.hosting.windowsservices | 7.0.0
microsoft.extensions.http | 7.0.0
microsoft.extensions.http.polly | 7.0.4
microsoft.extensions.logging | 7.0.0
microsoft.extensions.logging.abstractions | 7.0.0
microsoft.extensions.logging.console | 7.0.0
microsoft.extensions.options | 7.0.1
microsoft.extensions.options.configurationextensions | 7.0.0
microsoft.jquery.unobtrusive.ajax | 3.2.6
microsoft.owin | 4.2.2
microsoft.owin.host.systemweb | 4.2.2
microsoft.owin.security.cookies | 4.2.2
microsoft.owin.security.oauth | 4.2.2
microsoft.visualstudio.web.codegeneration.design | 7.0.5
system.componentmodel.annotations | 5.0.0
system.io.compression.zipfile | 4.3.0
system.net.http | 4.3.4
system.servicemodel.http | 4.10.2
system.servicemodel.primitives | 4.10.2
system.text.encoding.codepages | 7.0.0



</body>

</html>

**Comment by Tratcher** [comment]
TODO:
- Filter out the obsolete packages like https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Core
- Some of these are produced from other repose (like Microsoft.Owin). We'll file issues there and link back.
- Make assignments by feature area.

**Comment by mkArtakMSFT** [comment]
Moving the tracking issue to the GA milestone, as we physically won't be able to complete this work in rc2, even though it is going to continue.

**Comment by MackinnonBuck** [comment]
I've added READMEs for the non-shared-framework packages produced by this repo. The table in https://github.com/dotnet/aspnetcore/issues/48392#issuecomment-1597715952 has been updated to reflect that.

My criteria for skipping a package was:
* It's included in the ASP.NET shared framework,
* or we haven't produced a new version in a while,
* or the package is "obsolete" (bugfixes only, docs removed in .NET 8, etc.).

I'm open to going back and adding READMEs for shared framework packages if we think it's worthwhile to do so.
